cpp/ps0003/ps0003_sol.cpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+int main()
+{
+    // prompting user for number 2/2
+    int number;
+    std::cout << "please enter the number\n";
+    std::cin >> number;
+    
+    // checking if the number is odd or even 3/3
+    if (number % 2 == 0) 
+        std::cout << "the number is even\n";   
+    else 
+        std::cout << "the number is odd\n";
```

Comment:
- [x] used modulo operation by 2 `%2` to check if the number is even or odd
- [x] number is even when modulo result returns `0` and odd when returns `1`
- [x] used `std::cout` in a right way to print result in console